schemas/iso19139/src/main/plugin/iso19139/index-fields/index-subtemplate.xsl

```diff
@@ -151,8 +149,6 @@
                         else concat(gmd:name/gco:CharacterString, ' ', gmd:version/gco:CharacterString)"/>
     <resourceTitle><xsl:value-of select="$title"/></resourceTitle>
 
-    <any><xsl:value-of select="$title"/></any>
```

Comment:
It avoid failure when indexing... but I have got to check that extents (also formats and contacts) are still queryable/searchable in the admin ui (manage directories).

Reply:
Yes, it seems we need a multilingual any index for extent to be searchable (can't see any format at this point in directories browser, wonder why).